script/openqa-clone-custom-git-refspec

```diff
@@ -39,4 +39,5 @@ build="${build:-"$repo_name#$pr"}"
 casedir="${casedir:-"$repo#$branch"}"
 GROUP="${GROUP:-0}"
 dry_run="${dry_run:-""}"
-$dry_run openqa-clone-job --skip-chained-deps --within-instance $host $job _GROUP=$GROUP TEST=$test BUILD=$build CASEDIR=$casedir PRODUCTDIR=$productdir NEEDLES_DIR=$needles_dir
+shift 2 # eat the two params so that $@ contains only test vars
```

Comment:
would you mind using the POSIX sh compatible `"${@:3}"` as last parameter to `openqa-clone-job` in the next line instead as in
https://github.com/okurz/scripts/blob/master/openqa-clone-set#L19

Reply:
Done